frameworks/Crystal/kemal/shard.yml

```diff
@@ -4,15 +4,12 @@ version: "0.0.1"
 dependencies:
   pg:
     github: "will/crystal-pg"
-    version: "0.9.0"
-  pool:
-    github: "ysbaddaden/pool"
-    version: "0.2.3"
+    version: "0.13.3"
   kemal:
     github: "sdogruyol/kemal"
-    version: "0.16.0"
+    branch: master
```

Comment:
Everything here looks good, but we need kemal tied to a specific commit or release.